MONGODB/F1.2.md

```diff
@@ -0,0 +1,37 @@
+# F1.1 An Introduction to MongoDB and Atlas
+
+MongoDB is a database program which is used for high-volume data storage. Databases allow for the storage and retrieval of data. 
+
+A database is needed for dynamic websites, as they rely on data in order to function. Examples of dynamic websites include Quora, and Gmail, as they both store data and retrieve it. Static websites such as personal websites, do not need databases because they do not have to retieve any data. 
+
+MongoDB has three distinct components, each with their own unique tools: Atlas, Compass, and Stitch.
+
+## Atlas
+
+MongoDB Atlas handles the deployment of your database, using the cloud provider of your choice. Let's get started.
+
+First, sign up for MongoDB on this site: "https://www.mongodb.com/cloud/atlas?tck=docs_atlas"
+
+This is how the webpage should look:
+
+![Screen Shot 2020-03-27 at 4.19.56 AM](/Users/vishnuv/Library/Application Support/typora-user-images/Screen Shot 2020-03-27 at 4.19.56 AM.png)
+
+
+
+Hit the "Try Free" button and sign up. Your screen should then look like this: 
+
+
+
+![Screen Shot 2020-03-27 at 5.01.46 AM](/Users/vishnuv/Library/Application Support/typora-user-images/Screen Shot 2020-03-27 at 5.01.46 AM.png)
+
+ Choose the "Starter Clusters" Option and Create a cluster! A cluster is essentially a collection of various nodes, which each handle different parts of data. The main advantage that comes with using a cluster is that nodes are mainly independent of each other and can be examined seperately. 
```

Comment:
explain what a node is? Explain collections. Explain what a NoSQL database and why it is important